language/tools/move-cli/src/main.rs

```diff
@@ -781,9 +782,12 @@ fn view(args: &Move, file: &str) -> Result<()> {
 /// (1) all modules pass the bytecode verifier
 /// (2) all modules pass the linker
 /// (3) all resources can be deserialized
-/// (4) all events can be deserialized (TODO)
-/// (5) build/mv_interfaces is consistent with the global storage (TODO)
+/// (4) all events can be deserialized
+/// (5) build/mv_interfaces is consistent with the global storage (TODO?)
```

Comment:
I think #7010 should ensure that `build/mv_interfaces` is consistent with `storage`, by the fact that every time a the storage `save_modules()`, `generate_interface_files()` are invoked to update the content in `mv_interfaces`. And as suggested, another PR will make it less imperative and impossible to forget to do the interface generation upon module saving.